src/components/media_manager/src/media_manager_impl.cc

```diff
@@ -298,13 +298,29 @@ void MediaManagerImpl::StopStreaming(
     int32_t application_key, protocol_handler::ServiceType service_type) {
   SDL_LOG_AUTO_TRACE();
 
-  stream_data_size_ = 0ull;
-
   if (streamer_[service_type]) {
     streamer_[service_type]->StopActivity(application_key);
   }
 }
 
+void MediaManagerImpl::SuspendStreaming(
+    int32_t application_key, protocol_handler::ServiceType service_type) {
+  SDL_LOG_AUTO_TRACE();
+  using namespace application_manager;
+  using namespace protocol_handler;
+
+  ApplicationSharedPtr app = application_manager_.application(application_key);
+  if (app) {
```

Comment:
@AKalinich-Luxoft 
Maybe const auto app

Reply:
@SNiukalov fixed in ce5bce1528d56f60e65492288d1f6d1c1f1b9372